central/group/datastore/datastore_test.go

```diff
@@ -396,11 +399,19 @@ func (s *groupDataStoreTestSuite) TestAllowsMutate() {
 		Key:            "123",
 		Value:          "123",
 	}, RoleName: "123"}
-	err = s.dataStore.Mutate(s.hasWriteAccessCtx, []*storage.Group{groupWithID}, []*storage.Group{groupWithID},
+	err = s.dataStore.Mutate(s.hasWriteAccessCtx, false, []*storage.Group{groupWithID}, []*storage.Group{groupWithID},
 		[]*storage.Group{grp})
 	s.NoError(err, "expected no error trying to write with Access permissions")
 }
 
+func fixtureGroupWithMutability(mode storage.MutabilityMode) *storage.Group {
```

Comment:
I don't think we should keep this function within here, but instead add it to `pkg/fixtures/group.go`.
This way, in case other tests will require this for testing purposes, we have it in a shared place.